### PR TITLE
Add new relationship graphs in the detail pages.

### PR DESCRIPTION
**Pull request by karanpopat**
### New Dashboards
- [x] RDS Instance Dashboard
- [x] RDS Instance Detail
- [x] RDS Instance Age Report
- [x] RDS Instance Public Access Report
- [x] VPC VSwitch Detail
- [x] RAM Policy Detail
- [x] ECS Snapshot Detail

### CODE REVIEW

File: dashboards/vpc/vpc_vswitch_detail.sp [hunk 1]
```diff
@@ -0,0 +1,492 @@
+dashboard "vpc_vswitch_detail" {
+
+  title         = "Alicloud VPC Vswitch Detail"
```
Comment (rajlearner17) [inline dashboards/vpc/vpc_vswitch_detail.sp:3]:
```suggestion
  title         = "Alicloud VPC vSwitch Detail"
```

File: dashboards/rds/rds_instance_public_access_report.sp [hunk 2]
```diff
@@ -0,0 +1,62 @@
+dashboard "rds_instance_public_access_report" {
+
+  title         = "Alicloud RDS Instance Public Access Report"
```
Comment (misraved) [inline dashboards/rds/rds_instance_public_access_report.sp:3]:
```suggestion
  title         = "AliCloud RDS Instance Public Access Report"
```

File: dashboards/rds/rds_instance_report_age.sp [hunk 3]
```diff
@@ -0,0 +1,147 @@
+dashboard "rds_instance_age_report" {
+
+  title         = "Alicloud RDS Instance Age Report"
```
Comment (misraved) [inline dashboards/rds/rds_instance_report_age.sp:3]:
```suggestion
  title         = "AliCloud RDS Instance Age Report"
```

File: dashboards/vpc/vpc_vswitch_detail.sp [hunk 4]
```diff
@@ -0,0 +1,492 @@
+dashboard "vpc_vswitch_detail" {
+
+  title         = "Alicloud VPC vSwitch Detail"
```
Comment (misraved) [inline dashboards/vpc/vpc_vswitch_detail.sp:3]:
```suggestion
  title         = "AliCloud VPC vSwitch Detail"
```

**Review by misraved — COMMENTED** [review]
Please take a look at the review comments!!

**Review by misraved — COMMENTED** [review]
Please take a look at the review comments!!

File: dashboards/ecs/ecs_disk_detail.sp [hunk 5]
```diff
@@ -209,11 +374,24 @@ query "alicloud_ecs_disk_encryption" {
...
   EOQ
+}
 
-  param "arn" {}
+query "ecs_disk_auto_snapshot" {
+  sql = <<-EOQ
+    select
+      'Auto Snapshot' as label,
+      case when enable_auto_snapshot then 'Enabled' else 'Disabled' end as value,
+      case when enable_auto_snapshot then 'ok' else 'alert' end as type
```
[the remark this answers is not in the document]
Comment (bigdatasourav) [reply dashboards/ecs/ecs_disk_detail.sp:384]: Enabled and Disabled as values

**Review by misraved — APPROVED** [review]

**Pull request merged**